### DISCUSSION TITLE
服务器端、客户端部署在同一个境外VPS上的疑问

### DISCUSSION THREAD

**Discussion by jjyou-github** [enfein/mieru#26] [Q&A]
如题部署，国内电脑直接设置vps客户端的ip和端口的socks代理即可访问
问题是：
socks代理没有加密，是不是等于我招摇过市的告诉GFW我要通过代理上网了？

**Comment by enfein** [comment]
socks5 客户端，例如 chrome 浏览器，和 mieru 客户端之间的通信是没有加密的。

正常的做法是 mieru 客户端放在墙内的本地电脑上，然后 mieru 客户端和 mita 服务器端之间建立加密的连接。

你为什么要把 mieru 客户端也放在墙外 VPS 里面运行？这种做法没有任何好处。

**Comment by enfein** [comment]
Issue #19 实现之后，可以设定只有 localhost 才能与 mieru 客户端建立连接，进一步保护代理不被别人白嫖。
